raiden-network/light-client
# dapp: Introduces default monitoring configuration for tokens

Fixes #1791  

**Short description**
As part of the introduction of the new configuration, I took the time to unify our raiden-package configuration into one file instead.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Press connect
2. Ensure that you get monitored events for   `0x59105441977ecD9d805A4f5b060E34676F50F806`, `0xC563388e2e2fdD422166eD5E76971D11eD37A466`
3. Select a new token in the SelectHub screen, ensure that a monitored action is fired
